app/components/sidebars/main/channels_list/switch_teams_button/switch_teams_button.js

```diff
@@ -46,13 +46,20 @@ export default class SwitchTeamsButton extends React.PureComponent {
 
         const styles = getStyleSheet(theme);
 
+        const lowMentionCount = mentionCount <= 0;
+        const minWidth = lowMentionCount ? 8 : 20;
+        const maxWidth = mentionCount >= 99 ? 30 : 26;
```

Comment:
same as my other comment

Reply:
@enahum do you want me to do the same with minWidth or are those styles okay?

Reply:
I think minWidth is ok as if needed the width grows and does not cutoff content